app/src/main/java/org/akvo/flow/util/InternetDataConnection.java

```diff
@@ -0,0 +1,325 @@
+/*
+* Copyright (C) 2010-2017 Stichting Akvo (Akvo Foundation)
```

Comment:
FYI - if you are applying the copyright to a file, make sure the year matches when that source file was edited.  So in this case, the file is first created in 2017 so copyright should start in 2017.